pkg/providers/nutanix/template.go

```diff
@@ -239,6 +239,10 @@ func buildTemplateMapCP(
 		values["additionalCategories"] = controlPlaneMachineSpec.AdditionalCategories
 	}
 
+	if controlPlaneMachineSpec.BootType == v1alpha1.NutanixBootTypeLegacy || controlPlaneMachineSpec.BootType == v1alpha1.NutanixBootTypeUEFI {
```

Comment:
If we already have a [validation](https://github.com/aws/eks-anywhere/pull/9714/files#diff-348864fa8270a3c83b58035be1472204c447c0bc8279399180ede512ad949b0aR203-R206) for the specified bootType in Machine Cfg object, i think it's safe to do 
```suggestion
	if controlPlaneMachineSpec.BootType != "" {
	  values["bootType"] = controlPlaneMachineSpec.BootType
	}
```